components/brave_wallet_ui/page/screens/swap/components/swap/quote-info/quote-info.tsx

```diff
@@ -359,6 +365,13 @@ export const QuoteInfo = (props: Props) => {
   ])
 
   const effectiveFeeAmount = braveFee && new Amount(braveFee.effectiveFeePct)
+  const firstRoutesName = selectedQuoteOption?.sources[0]?.name ?? ''
```

Comment:
I think you meant `firstStepName`? I suggest picking the step that has type `kCross` in case of multiple steps. In case of single step, just use the one.

Reply:
Done!